bindings/matrix-sdk-ffi/src/app.rs

```diff
@@ -70,6 +72,10 @@ impl App {
         })))
     }
 
+    pub fn current_state(&self) -> AppState {
+        self.inner.observe_state().get().into()
+    }
```

Comment:
Maybe instead of an extra method we should send the current state immediately when subscribing? Like done [here](https://github.com/matrix-org/matrix-rust-sdk/pull/2245/files).

Reply:
Yep, I expect that they be used at different times, so I'll keep it like this :+1: